spyderlib/plugins/runconfig.py

```diff
@@ -467,8 +474,15 @@ def setup_page(self):
         interpreter_layout.addWidget(self.dedicated_radio)
         interpreter_layout.addWidget(self.systerm_radio)
         
-        wdir_group = QGroupBox(_("Working directory"))
-        wdir_bg = QButtonGroup(wdir_group)
+        general_group = QGroupBox("General settings")
+        command_options_enabled = self.create_checkbox(_("Command line options:"),

```

Comment:
These are not command line args for the Python interpreter but for the script you're running. For example:

    --number 10 --length 0.5 -I 12

That's why i think it doesn't make sense to let users set them globally. In fact, this could cause profound misunderstandings, so I'm really -1 to expose them in Preferences.